generated/nidcpower/nidcpower.proto

```diff
@@ -511,19 +431,92 @@ enum SourceMode {
   SOURCE_MODE_NIDCPOWER_VAL_SEQUENCE = 1021;
 }
 
-enum TransientResponse {
-  TRANSIENT_RESPONSE_UNSPECIFIED = 0;
-  TRANSIENT_RESPONSE_NIDCPOWER_VAL_NORMAL = 1038;
-  TRANSIENT_RESPONSE_NIDCPOWER_VAL_FAST = 1039;
-  TRANSIENT_RESPONSE_NIDCPOWER_VAL_SLOW = 1041;
-  TRANSIENT_RESPONSE_NIDCPOWER_VAL_CUSTOM = 1042;
-}
 
-enum TriggerType {
-  TRIGGER_TYPE_UNSPECIFIED = 0;
-  TRIGGER_TYPE_NIDCPOWER_VAL_NONE = 1012;
-  TRIGGER_TYPE_NIDCPOWER_VAL_DIGITAL_EDGE = 1014;
-  TRIGGER_TYPE_NIDCPOWER_VAL_SOFTWARE_EDGE = 1015;
+enum NiDCPowerViInt32Enum{
+  option allow_alias = true;
+  NI_D_C_POWER_VI_INT32_ENUM_AUTORANGE_BEHAVIOR_UNSPECIFIED = 0;
```

Comment:
We need to find a way to make the prefix look better: `NI_DCPOWER_` or `NIDCPOWER_